### PR TITLE
Restrict code ownership for strings

### PR DESCRIPTION
**Pull request by BenHenning**
## Explanation
This restricts code ownership for strings since now that we have translations coming in (#3687) we need to make sure that strings are consistent, correct per localization requirements, have descriptions, and are well-written. Some of these will become static analysis checks in the future, but the onus is currently mainly on reviewers to ensure that strings are being added properly.

## Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by BenHenning** [comment]
I can't find conclusive instructions that this is the correct way to match string files in multiple directories, but it does work locally via .gitignore.

@rt4914 please let me know if you prefer retaining owners for strings, but I figured this would benefit you by being one less thing to review that seems to come up in most UI PRs today.

PTAL @rt4914 & @seanlip

**Review by seanlip — APPROVED** [review]
Thanks Ben, LGTM

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -54,6 +54,9 @@ gradlew.bat @BenHenning
 /app/src/main/res/**/*.xml @rt4914
 /utility/src/main/res/**/*.xml @rt4914
 
+# App UI strings.
+/app/src/main/res/values*/strings.xml @BenHenning
```
Comment (seanlip) [inline .github/CODEOWNERS:58]: Btw, to your question -- I think it is probably fine. This doc says glob patterns here follow gitignore syntax, with some exceptions: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners#codeowners-syntax -- and this doc lists the exceptions: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners#syntax-exceptions.

If it doesn't work out, perhaps having an additional subdirectory like /app/src/main/res/values/ar might be a reasonable alternative?

**Review by rt4914 — APPROVED** [review]
@BenHenning Lets shift the ownership to you for now and we will change it back to me if needed in future. Thanks

File: .github/CODEOWNERS (hunk 1 above)
Comment (BenHenning) [reply .github/CODEOWNERS:58]: Having each subdirectory is a bit annoying, but it'll work as a backup if we remove the glob xml patterns (which is basically just a bandaid over the current situation of trying to wrangle the large number of res directories).

I came to the same conclusion as you re: compatibility, but I vaguely remember trying something like this in the past & it not working. We'll see if it does this time.

**Comment by BenHenning** [comment]
Thanks both. Merging now.

**Comment by BenHenning** [comment]
Looks like this pattern is working correctly: #3687. This is a great test since it should let us simplify some of the resource directory codeowners moving forward.

**Comment by seanlip** [comment]
Great! That's awesome to hear :)

**Pull request merged**